man/assign.Rd

```diff
@@ -18,6 +18,9 @@
 #            LHS2 = RHS2,
 #            ...), by = ...]
 
+# 3. Multiple columns in place
+# DT[i, names(.SD) := lapply(.SD, fx), by = ...]
```

Comment:
example should use `.SDcols`, right? or at least mention it

Reply:
unless it means all columns, but better to provide the argument to make example more complete